package.json

```diff
@@ -36,7 +36,9 @@
     "canary:bumpVersion": "yarn lerna version 2.0.0-alpha.`git rev-parse --short HEAD` --exact --no-push --yes",
     "canary:publish": "yarn lerna publish from-package --dist-tag canary --yes --no-verify-access",
     "changelog": "lerna-changelog",
-    "postinstall": "yarn lock:update && yarn build:packages",
+    "postinstall": "run-p postinstall:**",
+    "postinstall:main": "yarn lock:update && yarn build:packages",
+    "postinstall:dev": "is-ci || husky install",
```

Comment:
Upgrade Husky (see https://dev.to/typicode/what-s-new-in-husky-5-32g5 for more details)